tests/resources/OmniFixture-dump.sql

```diff
@@ -216,6 +216,8 @@ CREATE TABLE `goal` (
   `revenue` float NOT NULL,
   `deleted` tinyint(4) NOT NULL DEFAULT '0',
   `event_value_as_revenue` tinyint(4) NOT NULL DEFAULT '0',
+  `ts_created` TIMESTAMP NULL,
```

Comment:
Changes to this files should not be done manually and are not needed. Database updates will be applied automatically after the OmniFixture is set up.
To recreate the OmniFixture sql, there is a command, but running that one will currently cause a lot other changes in the results.